agreement_legal/views/agreement.xml

```diff
@@ -33,6 +33,12 @@
                         class="oe_highlight"
                         attrs="{'invisible': [('state', '=', 'active')]}"
                     />
+                    <button
+                        string="Extend"
```

Comment:
I don't think 'Extension' is a good term as it does nothing more than copying.

Why not simply 'Copy' ?